client/components/admin/UpdateEmployee/UpdateEmployee.tsx

```diff
@@ -33,29 +36,52 @@ const UpdateEmployee: React.FC<EmployeeProps> = ({ list }: EmployeeProps) => {
     pin: updatedPin,
     first_name: firstName,
     last_name: lastName,
-    phone: updatedPhone,
+    phone: updatedCleanedPhone,
     email: updatedEmail,
     hourly_rate: hourlyRate
   };
 
-  function handleSubmit (){
-    fetch(`/api/employees/${pin}`, {
-      method: 'PUT',
-      headers: { 'Content-Type': 'application/json' },
-      body: JSON.stringify(body)
-    })
-    .then(res => res.json())
-    .then(data => {
-      //console.log(data);
-    })
-    .catch(err => console.log('error:', err));
+     // Update the cleaned phone state when the input value changes
+     const handlePhoneChange = (value: string) => {
+      const cleaned = cleanPhoneNumber(value);
+      setPhone(value);
+      setCleanedPhone(cleaned);
+      console.log('cleaned:', cleaned)
+    };
 
-    setValidated(true);
-  }
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    event.stopPropagation();
+
+    const form = event.currentTarget;
+
+    // send a window.alert() if the phone number is invalid
+    if (updatedCleanedPhone === null) {
+      window.alert('Please enter a valid phone number');
```

Comment:
Can we add some [form validation](https://react-bootstrap.netlify.app/docs/forms/validation/) here instead of the window.alert?